templates/security/certifications.html

```diff
@@ -178,73 +218,52 @@ <h2>Ubuntu compliance &amp; hardening profiles</h2>
-      <table class="p-table--security-certifications">
-        <thead>
-          <th></th>
-          <th class="u-align--center">Ubuntu 16.04 LTS</th>
-          <th class="u-align--center">Ubuntu 18.04 LTS</th>
-          <th class="u-align--center">Ubuntu 20.04 LTS</th>
-        </thead>
-        <tbody>
-          <tr>
-            <td>
-              <a href="/security/fips">FIPS 140-2 Level 1 certification</a>: A US and Canada government cryptographic module certification of compliance with the FIPS 140 data protection standard. US agencies, their service providers or other institutions that comply with similar requirements (e.g., HIPAA, PCI-DSS) are required to comply with FIPS 140.
-            </td>
-            <td class="u-align--center">
-              <div class="u-sv2">
-                <i class="p-icon--success u-sv2">Yes: Tooling and automation</i>
-              </div>
-            </td>
-            <td class="u-align--center">
-              <div class="u-sv2">
-                <i class="p-icon--success">Yes: Tooling and automation</i>
-              </div>
-            </td>
-            <td class="u-align--center">
-              <div class="u-sv2">
-                <i class="p-icon--success">Yes: Tooling and automation</i>
-              </div>
-            </td>
-          </tr>
-          <tr>
-            <td>
-              <a href="/security/cc">Common Criteria, EAL2, an internationally accepted security certification</a>: A 3rd party attestation of the security mechanisms in the operating system. Ubuntu has a Common Criteria EAL2 certification recognized by CCRA and EU SOGIS members.
-            </td>
-            <td class="u-align--center">
-              <i class="p-icon--success">Yes</i>
-            </td>
-            <td class="u-align--center">
-              <i class="p-icon--success">Yes</i>
-            </td>
-            <td class="u-align--center"></td>
-          </tr>
-        </tbody>
-      </table>
-      <a href="/security/contact-us"
+      <div class="row p-strip--shallow p-divider  u-vertically-center">
+        <div class="col-6 col-medium-3">
+
+          <a href="/security/fips">FIPS 140-2 Level 1</a>
+          <h3 class="p-heading--5">These modules are NIST-certified:</h3>
+          <ul class="p-list">
+            <li class="p-list__item is-ticked">Ubuntu 20.04 LTS</li>
+            <li class="p-list__item is-ticked">Ubuntu 18.04 LTS</li>
+            <li class="p-list__item is-ticked">Ubuntu 16.04 LTS</li>
+          </ul>
+        </div>
+        <div class="col-6 col-medium-3 p-divider__block">
+          <a href="/blog/ubuntu-22-04-fips-140-3-modules-available-for-preview">FIPS 140-3 Level 1</a>
+          <h3 class="p-heading--5">
+            These modules have been assessed by a NIST-approved testing laboratory and are awaiting final certification by <a href="https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/modules-in-process-list">CMVP</a>
```

Comment:
```suggestion
            These modules have been assessed by a NIST-approved testing laboratory and are awaiting final certification by <a href="https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/modules-in-process-list">CMVP</a>:
```
Missing a colon here